qemu/tests/cfg/multi_host.cfg

```diff
@@ -78,12 +62,23 @@
                 # Migration properties
                 - @default:
                     type = migration_multi_host
+                    variants:
+                        - after_login_vm:
+                        - early_boot_vm:
+                            login_timeout = 420
+                            paused_after_start_vm = yes
+                            need_to_login = no
+                            preprocess_env = yes
+                            start_migration_timeout = random
+                            min_random_timeout = 5
+                            max_random_timeout = 20
                 - with_reboot:
-                    only early_boot_vm
-                    only mig_online
-                    timeout_random:
-                        min_random_timeout = 1
-                        max_random_timeout = 5
+                    login_timeout = 420
+                    paused_after_start_vm = yes
+                    preprocess_env = yes
+                    start_migration_timeout = random
```

Comment:
it's string, does it work?

                        wait = self.params.get("start_migration_timeout", 0)
                        logging.debug("Wait for migration %s seconds." %
                                      (wait))
                        time.sleep(int(wait))